contrib/babelfishpg_tsql/src/catalog.c

```diff
@@ -1583,6 +1583,7 @@ static void alter_guest_schema_for_db(const char *dbname);
 /* Helper function Rename BBF catalog update*/
 static void rename_view_update_bbf_catalog(RenameStmt *stmt);
 static void rename_procfunc_update_bbf_catalog(RenameStmt *stmt);
+static void rename_object_update_bbf_schema_permission_catalog(RenameStmt *stmt, const char *obj_type);
```

Comment:
Nit: Let argument be (stmt, stmt->renameType) and deal with the obj_type inside fuction is a better implementation.